.github/workflows/release.yml

```diff
@@ -82,7 +82,7 @@ jobs:
 
   linux-build:
     name: Build Linux binaries
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Should we run tests on the oldest or newest available runner? I have a preference for pinning the runner, whichever version we decide to use.